main.go

```diff
@@ -28,6 +28,7 @@ func main() {
 	var basepath = flag.String("basepath", ".", "The path for the project to todocheck. Defaults to current directory")
 	var cfgPath = flag.String("config", "", "The project configuration file to use. Will use the one from the basepath if not specified")
 	var format = flag.String("format", "standard", "The output format to use. Available formats - standard, json")
+	var verboseRequested = flag.Bool("verbose", false, "Verbose mode")
```

Comment:
Let's change the description to:
```
Make todocheck more talkative
```

similar to how curl defines their verbose flag:
```
$ curl --help | grep verbose
  --trace-time    Add time stamps to trace/verbose output
  -v, --verbose       Make the operation more talkative

```